src/js/components/Footer/stories/Sitemap.js

```diff
@@ -28,6 +28,8 @@ const FooterContent = () => (
 export const Sitemap = () => (
   // Uncomment <Grommet> lines when using outside of storybook
   // <Grommet theme={...}>
+  // Choosing White instead of brand since the brand
+  // color does not fits well with the blackish background
```

Comment:
```suggestion
```
Appreciate that you are documenting and communicating your decisions, however in this case I don't think we need this comment